### PR TITLE
fix(CAD): Document CAD flow events.

### LINKED ISSUE #1712: document Connect Another Device flow events.
This is a reminder to me to document the CAD flow events now that I've added them on the content server.

See https://github.com/mozilla/fxa-content-server/pull/4787

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #1712 

@philbooth - this isn't ready yet, but is this the general format you are going for?

### CODE REVIEW

File: docs/metrics-events.md [hunk 1]
```diff
@@ -350,6 +372,11 @@ to each of the table names mentioned above:
 
 ## Significant changes
 
+### Train 88
+ * [Flow events added for clicks on app store
+   install links](https://github.com/mozilla/fxa-content-server/pull/5113)
+   * flow.{viewName}.app-store.(android|ios)
```
Comment (philbooth) [inline docs/metrics-events.md]: Should this be `link.app-store`?

File: docs/metrics-events.md [hunk 2]
```diff
@@ -191,7 +198,8 @@ View name|Description
 `signin-unblock`|The sign-in unblock screen
 `choose-what-to-sync`|Choose what to Sync
 `connect-another-device`|Connect another device, phase 1
-`sms`|Connect another device, phase 2
+`sms`|Connect another device, phase 2, Send an SMS form
+`sms.sent`|Connect another device, phase 2, SMS sent
```
Comment (philbooth) [inline docs/metrics-events.md]: While you're in here, are you able to furnish any of the empty rows above with descriptions? I have difficulty knowing which view is which in many cases and wasn't able to put a meaningful description to all of them

**Comment by philbooth** [comment]
@shane-tomlinson, yep, looks fine to me.

**Comment by shane-tomlinson** [comment]
@philbooth - mind an r? on this? Feel free to make suggestions if something doesn't make sense.

**Comment by philbooth** [comment]
Thanks @shane-tomlinson!

**Pull request merged**